doc/Language/Interactive.org

```diff
@@ -0,0 +1,3 @@
+* Interaction of the Juvix Programming Language
+In many respects Juvix should
```

Comment:
should what?

What a cliffhanger!

Reply:
That is for you to fill in, and for me to find out!